library/core/src/result.rs

```diff
@@ -1233,7 +1233,8 @@ impl<T, E> Result<Option<T>, E> {
     /// ```
     #[inline]
     #[stable(feature = "transpose_result", since = "1.33.0")]
-    pub fn transpose(self) -> Option<Result<T, E>> {
+    #[rustc_const_unstable(feature = "const_result", reason = "newly added", issue = "82814")]
```

Comment:
We don't seem to usually have a `reason` for `rustc_const_unstable`, so no reason to make one up here.

Reply:
Fixed.